scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -351,26 +372,84 @@ private void initFooter() {
         rootPage.addMember(selectWfLayout);
     }
 
-    private void fillVarsPart() {
-
+    private void fillVarsPart(String jobDescriptor) {
         DynamicForm variablesVisualForm = initVariablesVisualForm();
-
         Layout hiddenVarsLayout = initVariablesActualForm();
-
         initVarsLayout();
+        getGenericInformationAttributes(jobDescriptor);
+        Widget worfklowMetaDataWidget = prepareWorlflowInformationWidget();
+        varsLayout.addMember(worfklowMetaDataWidget);
         varsLayout.addMember(variablesVisualForm);
         varsLayout.addMember(hiddenVarsLayout);
         rootPage.addMember(varsLayout);
         rootPage.reflow();
     }
 
+    private Widget prepareWorlflowInformationWidget() {
+        return new HTML("<center> <img src=" + icon + " height=40px width=40px> <br> <b>Documentation :</b> <a href=" +
+                        documentation + " target=_blank>" + documentation + " </a><br> <b> Bucket name :</b> " +
+                        bucketName + " </center>");
+    }
+
+    private void getGenericInformationAttributes(String jobDescriptor) {
+        Document dom = XMLParser.parse(jobDescriptor);
+        NodeList genericInfo = dom.getElementsByTagName("genericInformation");
+        // check if the job has genericInformation or not
+        if (genericInfo != null && genericInfo.getLength() > 0) {
+            // get the first item
+            Node root = genericInfo.item(0);
+            NodeList genericInfoMetaDataList = root.getChildNodes();
+            for (int i = 0; i < genericInfoMetaDataList.getLength(); i++) {
+                Node genericInfoMetadata = genericInfoMetaDataList.item(i);
+                if (genericInfoMetadata.getNodeName().equals("info") && genericInfoMetadata.hasAttributes()) {
+                    NamedNodeMap attributes = genericInfoMetadata.getAttributes();
+                    getGenericInformationName(attributes);
+                }
+            }
+        }
+    }
+
+    private void getGenericInformationName(NamedNodeMap attributes) {
+        for (int j = 0; j < attributes.getLength(); j++) {
+            Node attribute = attributes.item(j);
+            if (attribute.getNodeType() == Node.ATTRIBUTE_NODE && attribute.getNodeName().equals("name")) {
+                if (attribute.getNodeValue().equalsIgnoreCase("bucketName"))
+                    existBucketName = true;
+                if (attribute.getNodeValue().equalsIgnoreCase("documentation"))
+                    existDocumentation = true;
+                if (attribute.getNodeValue().equalsIgnoreCase("workflow.icon"))
+                    existIcon = true;
+            }
+            getGenericInformationValue(attribute);
+        }
+    }
+
+    private void getGenericInformationValue(Node attribute) {
+        if (attribute.getNodeType() == Node.ATTRIBUTE_NODE && attribute.getNodeName().equals("value")) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "value" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)